protoc_docs/bin/py_docstring.py

```diff
@@ -53,5 +54,20 @@ def main(input_file=sys.stdin, output_file=sys.stdout):
                     docstring=struct.get_python_docstring(),
                 ),
             ))
+    for fn in _init_files(comment_data.keys()):
+        answer.append(CodeGeneratorResponse.File(
+            name=fn,
+            content='',
+        ))
     cgr = CodeGeneratorResponse(file=answer)
     output_file.write(cgr.SerializeToString())
+
+def _init_files(fns=[]):
```

Comment:
Do not use empty list as a default argument in Python. It creates a single instance which, if modified, would carry over from invocation to invocation.

Use empty tuple in this case.

Reply:
Oh wow. That kind of error looks terribly simple enough to be very common. Fixed.